src/blocks/RajMishra3/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "block_name": "Coolor Change",
```

Comment:
Same here.

Reply:
I thought writing Coolor instead of Color would be a little different so I did it.  If there is any problem please let me know I will rectify it